src/isWilsonPrime.js

```diff
@@ -20,6 +20,19 @@
  */
 function isWilsonPrime(num) {
   // write code here
+  if (num < 0 || num === 0 || num - Math.trunc(num) > 0) {
```

Comment:
this code can be simplify
```suggestion
  if (num <= 0 || Number.isInteger(num)) {
```